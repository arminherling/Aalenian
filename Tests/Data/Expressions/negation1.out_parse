AssignmentStatement: {
    Left: {
        DiscardLiteral: _
    }
    Right: {
        UnaryExpression: {
            Operation: Negation
            Expression: {
                NumberLiteral: 1
            }
        }
    }
}
