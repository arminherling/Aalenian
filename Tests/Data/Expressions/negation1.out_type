TypedAssignmentStatement: {
    Left: {
        Discard: {
            Type: _
        }
    }
    Right: {
        TypedNegationExpression: {
            Type: i32
            Expression: {
                I32Literal: {
                    Value: 1
                    Type: i32
                }
            }
        }
    }
    Type: i32
}
