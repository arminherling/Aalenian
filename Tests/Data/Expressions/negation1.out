AssignmentStatement: {
    Left: {
        Discard: _
    }
    Right: {
        UnaryExpression: {
            Operation: Negation
            Expression: {
                Number: 1
            }
        }
    }
}
